### ISSUE TITLE
Layout enhancements 

### ISSUE THREAD

**Issue by jeremysmartt** [Teradata/covalent#1707]
Resizability
Toolbar animations
Navigation animations

**Comment by ineselmufti** [comment]
hello, can I work on it.

**Comment by kriswinbush** [comment]
Absolutely, we encourage community development. Just submit a PR and will have a look at it.

**Comment by ineselmufti** [comment]
I'm trying to commit, however I'm getting this error 
'js' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
husky > pre-commit hook failed (add --no-verify to bypass)

do you have any ideas about it please.
